pages/dashboard.tsx

```diff
@@ -0,0 +1,335 @@
+              );
+            }
+            return buttons;
+          })()}
+          <Button type="text" disabled={page >= trades.length / 10}>
+            Next
+          </Button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default function dashboard() {
+  const trades = [
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.seller,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.seller,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+  ];
+
+  return (
+    <>
+      <Layout>
+        <div className="font-mono py-12 px-12 grid bg-gray-100 md:grid-cols-4 md:grid-rows-5 grid-cols-1 gap-4">
+          <div className="text-sm p-6 bg-white text-primary shadow-md md:row-start-1 md:row-span-5 flex flex-col space-y-4">
+            <div className="text-center flex flex-col  items-center w-full">
+              <img
+                src="http://tinygraphs.com/labs/isogrids/hexa16/TNB Crow?theme=seascape&numcolors=4&size=220&fmt=svg"
```

Comment:
use https:// where possible

Reply:
I changed it but they don't support https.